dev/change-scala-version.sh

```diff
@@ -19,7 +19,7 @@
 
 set -e
 
-VALID_VERSIONS=( 2.12 2.13 )
+VALID_VERSIONS=( 2.13 )
```

Comment:
No further modifications were made to this file, just made Scala-2.12 an invalid option. We can refactor this script in the future when we need to support multiple Scala versions again.